src/hivpy/column_names.py

```diff
@@ -167,13 +167,12 @@
 RESISTANCE_MUTATIONS = "resistance_mutations"   # int: total number of resistance mutations in an individual
 RTTA_MUTATIONS = "rttams"                       # int: number of RT (reverse transcriptase) gene thymidine analog mutations (TAMs)
 RT184_MUTATION = "rt184m"                       # bool: True if an individual has the RT gene M184 mutation
-RT65_MUTATION = "rt65m"                         # bool: True if an individual has the RT gene K65 mutation
 RT151_MUTATION = "rt151m"                       # bool: True if an individual has the RT gene Q151 mutation
+RT65_MUTATION = "rt65m"                         # bool: True if an individual has the RT gene K65 mutation
 RT103_MUTATION = "rt103m"                       # bool: True if an individual has the RT gene K103 mutation
 RT181_MUTATION = "rt181m"                       # bool: True if an individual has the RT gene Y181 mutation
 RT190_MUTATION = "rt190m"                       # bool: True if an individual has the RT gene G190 mutation
 PR32_MUTATION = "pr32m"                         # bool: True if an individual has the PR (protease) gene P32 mutation
-PR33_MUTATION = "pr33m"                         # bool: True if an individual has the PR gene P33 mutation
```

Comment:
I thought `PR33` was the only mutation missing from the `NEW RESISTANCE MUTATIONS ARISING (and dominating)` section, but upon further examination it seems `PR90` is also missing.

Reply:
Removing `PR90` for now, but it may be added as an atazanavir mutation at some point.